File: back/src/dtos/campus_dto.rs
use serde::{Deserialize, Serialize};

#[derive(Serialize, Deserialize)]
#[derive(Debug)]
pub struct CampusDTO {
    pub campus_id : Option<i32>,
    pub name : String,
    pub university_id: i32,
    pub state_id: i32
}

File: back-csharp/_dtos/CampusDto.cs
using System.Text.Json.Serialization;

namespace back_csharp._dtos;

public class CampusDto
{
    public int CampusId { get; set; }
    
    public string Name { get; set; } 
    
    public int UniversityId { get; set; } 
    
    public string UniversityName { get; set; } 
    
    public string StateName { get; set; } 
    
    public int StateId { get; set; } 
    
    public string ImgFile { get; set; } 
    
    public string ImgType { get; set; }

    public string FullFileName { get; set; } = null;
}